packages/babel-preset-fbjs/plugins/__tests__/__snapshots__/inline-requires-test.js.snap

```diff
@@ -10,6 +10,116 @@ var foo = require(\\"foo\\");
 "
 `;
 
+exports[`inline-requires does not transform require calls if its not needed: does not transform require calls if its not needed 1`] = `
+"
+function test () {
+  function require(condition) {
+    if (!condition) {
+      throw new Error('The condition is false');
+    }
+  }
+
+  require('test');
+}
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+function test() {
+  function require(condition) {
+    if (!condition) {
+      throw new Error('The condition is false');
+    }
+  }
+
+  require('test');
+}
+"
+`;
+
+exports[`inline-requires ensures that the inlined require still points to the global require function even if local require is not called: ensures that the inlined require still points to the global require function even if local require is not called 1`] = `
+"
+const foo = require('foo');
+
+function test() {
+  function require(condition) {
+    if (!condition) {
+      throw new Error('Condition is falsy');
+    }
+  }
+
+  foo.isOnline();
```

Comment:
Should this be 

```suggestion
  require(foo.isOnline());
```

To match the same test with `customStuff` and cover the case that the call-site of the "custom" `require` function is renamed?

I can see that the last test in this snapshot does exactly that. Do we need both tests?

Reply:
Yes, this is intentional and a different test as I wanted to test that it shouldn't rename the "require" function unnecessary